### PR TITLE
samples(healthcare v1): initial consent store CRUD method samples

### PR DESCRIPTION
**Pull request by noerog**
New directory and initial samples for Healthcare API consent modality. Part of b/224797364.

### CODE REVIEW

File: healthcare/consent/package.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  },
+  "scripts": {
+    "test": "mocha system-test/*.test.js --timeout=60000"
+  },
+  "devDependencies": {
+    "mocha": "^9.0.0"
+  },
+  "dependencies": {
+    "@googleapis/healthcare": "^3.0.0",
+    "mocha-test": "^2.0.4",
```
Comment (alexander-fenster) [inline healthcare/consent/package.json:19]: why is `mocha-test` needed?

File: healthcare/consent/package.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    "uuid": "^8.0.0"
```
Comment (alexander-fenster) [inline healthcare/consent/package.json:20]: Since `uuid` is only used in tests, it should be a dev dependency.

**Review by alexander-fenster — APPROVED** [review]

File: healthcare/consent/package.json (hunk 2 above)
Comment (noerog) [reply healthcare/consent/package.json]: Done.

File: healthcare/consent/package.json (hunk 1 above)
Comment (noerog) [reply healthcare/consent/package.json]: May have added it by mistake. Removed.

**Pull request merged**